xml/System.Data.Odbc/OdbcParameter.xml

```diff
@@ -746,9 +746,18 @@ public void CreateOdbcParameter()
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the offset of the data within the column.</summary>
+        <value>The offset of the data within the column. The default value is zero.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+ The <xref:System.Data.Odbc.OdbcParameter.Offset%2A> property is used for binary and string types. For fixed length data types, the value of <xref:System.Data.Odbc.OdbcParameter.Size%2A> is ignored.
+
+ For nonstring data types and ANSI string data, the <xref:System.Data.Odbc.OdbcParameter.Size> property refers to the number of bytes. For Unicode string data, <xref:System.Data.O.DbParameter.Size> refers to the number of characters.
```

Comment:
The build threw another warning:

```
api/System.Data.Odbc.OdbcParameter.Offset.yml
[Warning] 1 invalid cross reference(s) "<xref:System.Data.O.DbParameter.Size>".
```

The namespace is malformed. What's the correct namespace?

Reply:
Oops sorry, fixed and also correct bad references to Size instead of Offset. Hopefully it's OK now.